resources/migrations/019_add_schema_column_to_table.yaml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+  - changeSet:
+      id: 19
+      author: camsaul
+      changes:
+        - addColumn:
+            tableName: metabase_table
+            columns:
+              - column:
+                  name: schema
+                  type: VARCHAR
```

Comment:
So the migration I wrote to add the data migrations table doesn't declare a length limit for its `VARCHAR` either:

```yaml
name: id
type: VARCHAR
```

@agilliland Does that work for you on MySQL databases? I didn't know this, but apparently the varchar length limit param is optional for both postgres and H2 but required for MySQL. Maybe Liquibase is a applying a default limit to the column.

At any rate, we should consider configuring a CircleCI instance to use MySQL as its backing DB if we're going to support it.